components/date-picker/index.tsx

```diff
@@ -44,7 +44,7 @@ export interface DatePickerProps extends PickerProps, SinglePickerProps {
     disabledSeconds?: () => [number, number],
   };
   onOpenChange?: (status: boolean) => void;
-  onOk?: () => void;
+  onOk?: (selectedTime: moment.Moment) => void;
```

Comment:
onOk is a callback function, base on [TS docs/Optional Parameters in Callbacks](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html):

> but there’s no need to make the parameter optional to accomplish this – it’s always legal to provide a callback that accepts fewer arguments.

No need to make `selectedTime` optional, you can try this: [playground](https://www.typescriptlang.org/play/index.html#src=const%20eat%20%3D%20(cb%3A%20(something%3A%20string)%20%3D%3E%20void)%20%3D%3E%20%7B%20%7D%0D%0Aconst%20something%20%3D%20()%20%3D%3E%20%7B%7D%0D%0A%0D%0Aeat(something))